app/src/js/components/locale.js

```diff
@@ -34,6 +34,7 @@ export const strings = new LocalizedStrings({
     logo: 'cumulus-logo.png',
     operations: 'Operations',
     pdrs: 'Pdrs',
+    reconciliation_reports: 'Reconciliation Reports',
```

Comment:
If we add it here do we need to have it in the gitc block as well?  I think so, but I'm not 100% certain.